client/src/MapContainer.jsx

```diff
@@ -4,12 +4,12 @@ import {Map, InfoWindow, Marker, GoogleApiWrapper} from 'google-maps-react';
 export var MapContainer = (props) => (
```

Comment:
(similar suggestion here with destructuring props!) again i know front-end isn't the main focus so no pressure